src/main/frontend/pipeline-console-view/pipeline-console/main/hooks/use-steps-poller.ts

```diff
@@ -30,14 +30,34 @@ export function useStepsPoller(props: RunPollerProps) {
 
   const updateStepConsoleOffset = useCallback(
     async (stepId: string, forceUpdate: boolean, startByte: number) => {
-      const stepBuffer = stepBuffers.get(stepId) ?? {
-        lines: [],
-        startByte: 0 - LOG_FETCH_SIZE,
-        endByte: -1,
-        stepId,
```

Comment:
Side note: `stepId` is not valid in a `StepLogBufferInfo` entry, hence I removed it.